### ISSUE TITLE
make server isn't working

### ISSUE THREAD

**Issue by Bharghvi** [AdeleD/react-paginate#157]

**Comment by AdeleD** [comment]
Did you mean `make serve`?

**Comment by mundosk8** [comment]
It is working fine, i think this issue should be closed. Also i tested right now the node 8 + npm 5 thing, and it works fine by now. Node v8.7.0, npm 5.5.1

**Comment by vinogradov** [comment]
it may not working because
```
  "peerDependencies": {
    "react": "^16.0.0"
  },
```

The thing is that `npm install` doesn't install peer dependencies. To fix this you may move `"react": "^16.0.0"` to `devDependencies` and rerun `npm install`

**Comment by erwstout** [comment]
Tried @vinogradov's suggestion and it still does not run.

```sh
make serve
node demo/server.js
Server running on development mode on port 3000
Hash: 382ae50d5b1ae957e524
Version: webpack 1.15.0
Time: 1921ms
      Asset     Size  Chunks             Chunk Names
    demo.js  1.02 MB       0  [emitted]  main
demo.js.map  1.21 MB       0  [emitted]  main
chunk    {0} demo.js, demo.js.map (main) 989 kB [rendered]
    [0] ./demo/js/demo.js 6.71 kB {0} [built] [1 error]
    [1] ./~/react-dom/index.js 1.36 kB {0} [built]
    [2] ./~/process/browser.js 5.42 kB {0} [built]
    [3] ./~/react-dom/cjs/react-dom.production.min.js 92.7 kB {0} [built] [1 error]
    [4] ./~/fbjs/lib/ExecutionEnvironment.js 935 bytes {0} [built]
    [5] ./~/object-assign/index.js 2.11 kB {0} [built]
    [6] ./~/fbjs/lib/emptyFunction.js 959 bytes {0} [built]
    [7] ./~/fbjs/lib/EventListener.js 2.25 kB {0} [built]
    [8] ./~/fbjs/lib/getActiveElement.js 912 bytes {0} [built]
    [9] ./~/fbjs/lib/shallowEqual.js 1.62 kB {0} [built]
   [10] ./~/fbjs/lib/containsNode.js 923 bytes {0} [built]
   [11] ./~/fbjs/lib/isTextNode.js 479 bytes {0} [built]
   [12] ./~/fbjs/lib/isNode.js 702 bytes {0} [built]
   [13] ./~/fbjs/lib/focusNode.js 578 bytes {0} [built]
   [14] ./~/fbjs/lib/emptyObject.js 332 bytes {0} [built]
   [15] ./~/react-dom/cjs/react-dom.development.js 547 kB {0} [built] [1 error]
   [16] ./~/fbjs/lib/invariant.js 1.51 kB {0} [built]
   [17] ./~/fbjs/lib/warning.js 1.9 kB {0} [built]
   [18] ./~/prop-types/checkPropTypes.js 2.87 kB {0} [built]
   [19] ./~/prop-types/lib/ReactPropTypesSecret.js 314 bytes {0} [built]
   [20] ./~/fbjs/lib/hyphenateStyleName.js 848 bytes {0} [built]
   [21] ./~/fbjs/lib/hyphenate.js 674 bytes {0} [built]
   [22] ./~/fbjs/lib/camelizeStyleName.js 875 bytes {0} [built]
   [23] ./~/fbjs/lib/camelize.js 582 bytes {0} [built]
   [24] ./react_components/index.js 286 bytes {0} [built]
   [25] ./react_components/PaginationBoxView.js 10.6 kB {0} [built] [1 error]
   [26] ./~/prop-types/index.js 956 bytes {0} [built]
   [27] ./~/prop-types/factoryWithTypeCheckers.js 19.8 kB {0} [built]
   [28] ./~/prop-types/factoryWithThrowingShims.js 1.49 kB {0} [built]
   [29] ./~/classnames/index.js 1.1 kB {0} [built]
   [30] ./~/react-addons-create-fragment/index.js 9.57 kB {0} [built] [1 error]
   [31] ./react_components/PageView.js 1.26 kB {0} [built] [1 error]
   [32] ./react_components/BreakView.js 512 bytes {0} [built] [1 error]
   [33] ./~/jquery/dist/jquery.js 268 kB {0} [built]

ERROR in ./demo/js/demo.js
Module not found: Error: Cannot resolve module 'react' in /Users/ericstout/Documents/projects/react-paginate/demo/js
 @ ./demo/js/demo.js 12:13-29

ERROR in ./react_components/PaginationBoxView.js
Module not found: Error: Cannot resolve module 'react' in /Users/ericstout/Documents/projects/react-paginate/react_components
 @ ./react_components/PaginationBoxView.js 9:13-29

ERROR in ./~/react-dom/cjs/react-dom.production.min.js
Module not found: Error: Cannot resolve module 'react' in /Users/ericstout/Documents/projects/react-paginate/node_modules/react-dom/cjs
 @ ./~/react-dom/cjs/react-dom.production.min.js 12:20-36

ERROR in ./~/react-dom/cjs/react-dom.development.js
Module not found: Error: Cannot resolve module 'react' in /Users/ericstout/Documents/projects/react-paginate/node_modules/react-dom/cjs
 @ ./~/react-dom/cjs/react-dom.development.js 16:12-28

ERROR in ./react_components/PageView.js
Module not found: Error: Cannot resolve module 'react' in /Users/ericstout/Documents/projects/react-paginate/react_components
 @ ./react_components/PageView.js 7:13-29

ERROR in ./react_components/BreakView.js
Module not found: Error: Cannot resolve module 'react' in /Users/ericstout/Documents/projects/react-paginate/react_components
 @ ./react_components/BreakView.js 7:13-29

ERROR in ./~/react-addons-create-fragment/index.js
Module not found: Error: Cannot resolve module 'react' in /Users/ericstout/Documents/projects/react-paginate/node_modules/react-addons-create-fragment
 @ ./~/react-addons-create-fragment/index.js 10:12-28
webpack: Failed to compile.
```
